### PR TITLE
Initial Gazebo Implementation of 2024 Robot URDF

### LINKED ISSUE #55: Bring up our 2024 robot inside of gazebo

### PR DESCRIPTION
**Pull request by Isopod00**
Ready for review!

### CODE REVIEW

File: src/drivetrain/drivetrain/drivetrain_node.py [hunk 1]
```diff
@@ -11,22 +11,28 @@
 
 # Import ROS 2 formatted message types
 from geometry_msgs.msg import Twist
+from std_msgs.msg import Float64
 
 # Import custom ROS 2 interfaces
 from rovr_interfaces.srv import Stop, Drive, MotorCommandSet
 from rovr_interfaces.msg import AbsoluteEncoders
 
+simulation = True
```
Comment (Isopod00) [inline src/drivetrain/drivetrain/drivetrain_node.py]: we don't need this variable here anymore right?

File: .gitmodules
Comment (pwmarshall) [inline .gitmodules:1]: wait but no more submodule?

File: src/drivetrain/drivetrain/drivetrain_node.py (hunk 1 above)
Comment (pwmarshall) [reply src/drivetrain/drivetrain/drivetrain_node.py]: yeah i have a commit removing it

File: config/drivetrain_config.yaml
Comment (pwmarshall) [inline config/drivetrain_config.yaml:1]: Wasn't sure if you wanted this to be True or false

Comment (Isopod00) [reply config/drivetrain_config.yaml:1]: Good point, can you please make it false by default

File: .gitmodules
Comment (Isopod00) [reply .gitmodules:1]: idk, are we keeping the ros_gz_moon submodule or not @jmblixt3? It's probably fine either way tbh

Comment (jmblixt3) [reply .gitmodules:1]: Unless we move peters code over

Comment (Isopod00) [reply .gitmodules:1]: The ros_gz_moon submodule has been removed and Peter's code was moved over

**Comment by pwmarshall** [comment]
to test on ubuntu (and generally linux)
with ros installed:
`sudo apt-get install ros-humble-ros-gz` to get gazebo
`sudo apt-get install ros-humble-teleop-twist-keyboard` to get keyboard control for ros

then run from local repo
```
colcon build --symlink-install --packages-up-to ros_gz_launch
source install/setup.bash
ros2 launch ros_gz_launch master_ASM.launch.py
```
```
colcon build --symlink-install --packages-up-to drivetrain
source install/setup.bash
ros2 run drivetrain drivetrain_node
```
and 
```
source /opt/ros/humble/setup.bash
ros2 run teleop_twist_keyboard teleop_twist_keyboard
```

These will launch Gazebo, DrivetrainNode, and keyboard controls in that order.

**Review by pwmarshall — APPROVED** [review]
Made it and tested, is good

File: README.md [hunk 2]
```diff
@@ -82,6 +87,11 @@ To normalize line endings in git, use the command:
 git config --global core.autocrlf true
 ```
 
+## Gazebo Installation
+Install Gazebo Harmonic [here](https://staging.gazebosim.org/docs/harmonic/install_ubuntu).
```
Comment (Isopod00) [inline README.md]: Change this since we are using Gazebo Fortress, not Harmonic

**Review by jmblixt3 — APPROVED** [review]

**Review by Isopod00 — COMMENTED** [review]
Looks good to me

**Pull request merged**